EventProcessor/EventProcessor.WorkerRole/settings.job

```diff
@@ -0,0 +1 @@
+{ "stopping_wait_time":  15 }
```

Comment:
Again, not sure we need this setting on this webjob, since we don't care if it shuts down.